src/components/Simulering/Simulering.tsx

```diff
@@ -193,12 +194,22 @@ export function Simulering(props: {
         <div id="alt-chart-title" hidden>
           <FormattedMessage id="beregning.alt_tekst" />
         </div>
-        <div data-testid="highcharts-aria-wrapper" aria-hidden={true}>
+        <div
+          className={styles['highcharts-wrapper']}
+          data-testid="highcharts-aria-wrapper"
+          aria-hidden={true}
+        >
           <HighchartsReact
             ref={chartRef}
             highcharts={Highcharts}
             options={chartOptions}
           />
+          <div className={styles['info-click']}>
+            <HandFingerIcon />
+            <BodyShort size="small">
+              <FormattedMessage id="beregning.highcharts.informasjon_klikk" />
+            </BodyShort>
+          </div>
```

Comment:
Forslag:
1. Vi kan forenkle koden ved å slå sammen div og BodyShort.
2. Vi kan sette farge på BodyShort med prop-en `textColor` slik at vi kan kvitte oss med litt CSS.
```suggestion
          <BodyShort
            size="small"
            textColor="subtle"
            className={styles['info-click']}
          >
            <HandFingerIcon />
            <FormattedMessage id="beregning.highcharts.informasjon_klikk" />
          </BodyShort>
```